components/progress-bar/src/overrides/bar.js

```diff
@@ -1,29 +1,42 @@
 /** @jsx jsx */
 
-import { jsx, useBrand } from '@westpac/core';
+import { jsx, useBrand, getLabel } from '@westpac/core';
 
-const Bar = ({ state, ...rest }) => <div {...rest} />;
+const Bar = ({ state: { value }, ...rest }) => <div style={{ width: `${value}%` }} {...rest} />;
```

Comment:
I would have put this into the main component and not into the overrides so that it's not overridden when someone want's a different tag.